script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -820,6 +820,8 @@ If that happened, I would turn into something even more horrifying.
 [sha:8b251ff93fe01a5fc3b929aaa37269e9ecaed252]{
 -- Page 40, Offset 33217.
 -- 「■■■■■。■■■■■■■■■■■、■■■■？」
+// Kinda annoying to fully decipher (can't hear the first part of the first clause)
+// Include TL as ruby?
 "■■■■■. ■■■■■■■■■■■, ■■■■?"
```

Comment:
>お機嫌はよろしいですか。お辛いでしたら、お声をかけてください。

Is everything in order? If you're feeling down, please feel free to let me know.

Reply:
There's also one earlier at Offset 33182
>志貴さん？　いかがなさいましたか。やはり、お機嫌は？

>Shiki-san? How are you holding up? You're not feeling well, are you?

Reply:
Maybe we could render these upside down for the trippy effect, or something.
Ruby feels too straightforward

Reply:
Thoughts?


![2023-05-31-230251_2160x1286_scrot](https://github.com/Tsukihimates/Tsukihime-Translation/assets/2476799/6b75368d-7d5b-4ad7-b4f3-db1b82b23a0b)